cmds/core/cat/cat.go

```diff
@@ -36,8 +37,20 @@ func catFile(w io.Writer, file string) error {
 	return err
 }
 
-func cat(w io.Writer, files []string) error {
-	for _, name := range files {
+var (
+	osStdin  io.ReadWriter = os.Stdin
+	osStdout io.ReadWriter = os.Stdout
```

Comment:
Instead of global variables, use the pattern like https://github.com/u-root/u-root/blob/master/cmds/contrib/spidev/spidev.go#L113